jsonschema-hyperschema.xml

```diff
@@ -390,8 +389,13 @@
 
             <section title="Links and data">
                 <t>
-                    "Form"-like functionality can be defined by use of the <xref target="method">"method"</xref> and <xref target="submissionSchema">"submissionSchema"</xref> keywords, which supplies a schema describing the data to supply to the server.
-                    Functionality equivalent to dynamic URI generation is available through the <xref target="href">"href"</xref> template and <xref target="hrefSchema">"hrefSchema"</xref>.
+                    "Form"-like functionality can be defined by use of the
```

Comment:
I don't think we can use the term "Form-like" anymore.  I think we can say this without using the word "form".